modules/statics/src/base.ts

```diff
@@ -176,7 +176,6 @@ export enum UnderlyingAsset {
   EUROC = 'euroc',
   GBP = 'gbp',
   GTC = 'gtc',
-  GTETH = 'gteth',
```

Comment:
Untracked breaking change missing in commit message footer.

Reply:
@mmcshinsky-bitgo Updated, Thanks for the nudge

Reply:
No problem :) Needs to be in the commit footer though :)

Reply:
@mmcshinsky-bitgo Updated, Kindly check, If its still not right?, Kindly give me the correct format